LanguageFeatures/Patterns/object_A02_t09.dart

```diff
@@ -0,0 +1,32 @@
+// Copyright (c) 2022, the Dart project authors.  Please see the AUTHORS file
+// for details. All rights reserved. Use of this source code is governed by a
+// BSD-style license that can be found in the LICENSE file.
+
+/// @assertion objectPattern ::= typeName typeArguments? '(' patternFields? ')'
+/// ...
+/// As with record patterns, the getter name can be omitted and inferred from
+/// the variable pattern in the field subpattern which may be wrapped in a unary
+/// pattern
+///
+/// @description Checks an object pattern with omitted getter name and a
+/// parenthesized subpattern in a declaration context
+/// @author sgrekhov22@gmail.com
+
+// SharedOptions=--enable-experiment=patterns
+
+import "../../Utils/expect.dart";
+import "patterns_lib.dart";
+
```

Comment:
This must be compile-time errors, cf. https://github.com/dart-lang/language/issues/2698 again.